conf/cluster.yaml.sample

```diff
@@ -18,7 +18,7 @@ cluster:
             - message_bus
       storage:
         - name: cvg-01
-          type: cas       # io (default), cas
+          type: ios       # io (default), cas
```

Comment:
comment here says 'io', shouldn't it be ios ?

Reply:
Yes, updated the comment.